helm/hpcc/values.schema.json

```diff
@@ -799,13 +799,19 @@
     },
     "eclccserver": {
       "type": "object",
-      "required": [ "name" ],
+      "required": [ "name", "maxActive" ],
       "additionalProperties": { "type": ["integer", "string", "boolean"] },
       "properties": {
         "name": {
           "type": "string",
           "description": "The name of the eclccserver process"
         },
+        "maxActive": {
+          "type": "integer",
+          "minimum": 1,
+          "description": "Maximum number of compile jobs that can be active simultaneously",
+          "default": 4
```

Comment:
That does this do?

Reply:
This mirrors the default in code (in eclccserver.cpp):
```
#ifdef _CONTAINERIZED
            queryCodeSigner().initForContainer();

            bool useChildProcesses = globals->getPropInt("@useChildProcesses", false);
            unsigned maxThreads = globals->getPropInt("@maxActive", 4);
#else

```

Reply:
Does it do anything though (do some editors use it?)  Genuine question...

Reply:
ah, no idea.